remix/app/helpers/localStorage.helper.ts

```diff
@@ -0,0 +1,18 @@
+import { Notification } from '../types';
+
+export function addNotification({ text, type }: Notification) {
+  localStorage.setItem('notificationText', text);
+  localStorage.setItem('notificationType', type as string);
+}
+
+export function getNotificationList() {
```

Comment:
Unless we decide to store more than one object of notifications (so an array of notificaitions), we could rename it to:

```suggestion
export function getNotification() {
```

Reply:
Improved here: [5b3237a](https://github.com/nimblehq/js-framework-benchmark/pull/120/commits/5b3237af21e9837710fef39f565836071a15e44b)